README.md

```diff
@@ -18,32 +18,44 @@
 
 ## Task 1
 
-There is a basic search component `patients_search.tsx`. It allows a user to type a query and have the results return to the parent component.
+There is a basic search feature in our app that allows search for patients based on several properties. The implementation of this feature has some problems. We're interested in two of them:
 
-It's a little janky, sometimes it doesn't search for the correct query. The component boundary is confusing also.
+1. There is a bug leading to incorrect results in some cirumstances. One simple way to cause it is typing "J" into the search after refreshing the page. It should show two search results, but nothing is displayed.
+2. The boundaries between the components in the implementation is convoluted.
 
 ### 1.1 Add a test
 
-Add a test for this component. Feel free to use whichever testing library you think is best.
+Add a test for the search, it should cover the bug described above and any other testcases you feel are important. Feel free to use whichever testing library you think is best.
```

Comment:
Maybe we want to rephrase the last sentence to `feel free to use whichever react testing library along with jest`

Reply:
I think if someone would actually rip out jest that would also be an interesting signal. I would at least want to know the reasoning for it